ui/src/views/Tasks/TaskIndex/ListNamespaces/index.jsx

```diff
@@ -136,12 +171,26 @@ export default class ListNamespaces extends Component {
     return (
       <Dashboard
         title="Task Index"
-        helpView={<HelpView description={description} />}>
+        helpView={<HelpView description={description} />}
+        search={
+          <Search
+            disabled={loading}
+            value={this.state.namespaceInput}
+            onChange={this.handleNamespaceInputChange}
+            onSubmit={this.handleTaskNamespaceSearchSubmit}
+            placeholder="Search"
+          />
+        }>
         <Fragment>
           {loading && <Spinner loading />}
           <ErrorPanel fixed error={namespacesError || taskNamespaceError} />
           {!loading && !hasNamespaces && !hasIndexedTasks && (
-            <Typography>No items for this page.</Typography>
+            <Redirect
+              to={`/tasks/index/${match.params.namespace
+                .split('.')
+                .slice(0, -1)
+                .join('.')}/${match.params.namespace.split('.').slice(-1)[0]}`}
+            />
```

Comment:
Navigating to `http://localhost:5080/tasks/index/yadi-yadi-yada` seems to redirect to `http://localhost:5080/tasks/index//yadi-yadi-yada` (double slash). That's probably a bug.